reasoner/controller/DisjunctionController.java

```diff
@@ -113,15 +113,12 @@ public void setUp() {
             }
         }
 
-        Stream<ConceptMap, ConceptMap> getOrCreateHubReactive(Stream<ConceptMap, ConceptMap> fanIn) {
-            // This method is only here to be overridden by root disjunction to avoid duplicating setUp
-            return fanIn;
-        }
+        abstract Stream<OUTPUT,OUTPUT> getOrCreateHubReactive(Stream<ConceptMap, ConceptMap> fanIn);
```

Comment:
Allows the ConditionController to do the transformation from ConceptMap to Either<...>